### ISSUE TITLE
Accessibility: Error and success messages have no icons

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#781]
Error and success messages do not have an icon indicating the state of the message for users that cannot use color information.

![image](https://user-images.githubusercontent.com/1659006/202415074-e0fb1208-12a3-4072-99ce-10cac5f07767.png)
_Checkboxes as viewed with protanopia (8 percent of men and 0.5 percent of women)_

Add checkmark and exclamation mark icons to error messages.

**Comment by gfellerph** [comment]
This is handled by a new design for valid states. Will create style update tickets when design is ready.
